### PR TITLE
fix for sample filenames less than 3 chars; clipboard supports multiple samples

### PR DESCRIPTION
**Pull request by defensem3ch**

### CODE REVIEW

File: src/main/java/kitEditor/KitEditor.java [hunk 1]
```diff
@@ -1188,9 +1189,9 @@ private void duplicateSample(Sample sample) {
     }
     
     private void pasteSample() {
-        for (Sample sample : clipboard) {
-            if (sample != null) {
-                duplicateSample(sample);
+        for (int index = 0; index < clipboard.length; ++index) {
+            if (clipboard[index] != null) {
+                duplicateSample(clipboard[index]);
```
Comment (jkotlinski) [inline src/main/java/kitEditor/KitEditor.java:1194]: what is the reason for this change?
was there some kind of merge problem?

**Comment by jkotlinski** [comment]
nice! could you add something to CHANGELOG.md to describe the change? thanks a lot!!

File: src/main/java/kitEditor/KitEditor.java (hunk 1 above)
Comment (defensem3ch) [reply src/main/java/kitEditor/KitEditor.java:1194]: yes, this change is so that multiple samples can be copied and pasted, rather than only one

**Comment by defensem3ch** [comment]
i've updated the changelog with the new features and fixes

**Review by jkotlinski — APPROVED** [review]
Excellent!
